wikimedia/wikipedia-ios
# Add UIComponents Package

**Phabricator:** 
https://phabricator.wikimedia.org/T329328

### Notes
This PR adds our new UIComponents package (repo [here](https://github.com/wikimedia/wikipedia-ios-ui-components)) to the project. The package is pretty much empty, aside from an empty Demo app project and a `UIComponents` struct (automatically added via New Package flow).

### Test Steps
1. On ExploreViewController.swift, add an `import UIComponents` to the top.
2. At the end of viewDidAppear, add a call to `print(UIComponents().text)`.
3. Run app, confirm you see "Hello World" printed to the console.
